devtron-labs/dashboard
# fix: System user locked.

# Description

After installing Devtron, initially 2 users are there. We have marked 'admin' user as locked.
Since, we cannot update permissions for 'system' users so it would be better to mark system user also as locked.

Ticket [AB1664](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=1664)
Fixes https://github.com/devtron-labs/devtron/issues/2832

## Type of change


# How Has This Been Tested?

- Navigate to global configurations
- Go to Authorization -> User Permission
- `system` user should be locked just as `admin`.


# Checklist:


<img width="1070" alt="Screenshot 2023-02-06 at 6 57 28 PM" src="https://user-images.githubusercontent.com/68757539/216983510-a5f458bf-70db-4d87-8111-65f93e7ed4d6.png">
